drivers/overlay/overlay.go

```diff
@@ -146,38 +178,45 @@ func InitWithName(name, home string, options []string, uidMaps, gidMaps []idtool
 		return nil, err
 	}
 
-	d := &Driver{
-		name:    name,
-		home:    home,
-		uidMaps: uidMaps,
-		gidMaps: gidMaps,
-		ctr:     graphdriver.NewRefCounter(graphdriver.NewFsChecker(graphdriver.FsMagicOverlay)),
-		opts:    opts,
+	supportsDType, err := fsutils.SupportsDType(home)
+	if err != nil {
+		return nil, err
+	}
+	if !supportsDType {
+		// not a fatal error until v17.12 (#27443)
```

Comment:
When do we intend to make this a fatal error?

Reply:
I would figure when moby does?  Or should we do it now?

Reply:
Does xfs support DType?  If we turn this on now, will it break on RHEL.  Man pages do not mention DT_TYPE for xfs. @rhvgoyal WDYT?

Reply:
This should be equivalent to whether or not the XFS filesystem was formatted using `ftype=1`.